### DISCUSSION TITLE
ZFP 1.0+ on conda-forge

### DISCUSSION THREAD

**Discussion by agolovanov** [ornladios/ADIOS2#4417] [Q&A]
I was trying to install the latest `adios2` in a new conda environment (from conda-forge) and encountered a version conflict with already installed `imagecodecs`. The conflict ultimately boils down to `imagecodecs` wanting `zfp >=1.0.1`, while `libadios2` required `zfp >=0.5.5,<1.0a0`.

As far as I can tell, it's not a fundamental limitation (the support of zfp version 1.0+ was added by https://github.com/ornladios/ADIOS2/pull/3312). So I guess the problem must be with an outdated `zfp 0.5` requirement in the conda-forge feedstock (https://github.com/conda-forge/adios2-feedstock/blob/main/recipe/meta.yaml). I've created an issue (https://github.com/conda-forge/adios2-feedstock/issues/105) on the feedstock. Is it the right place to report such problems?

**Comment by vicentebolea** [comment]
This has been solved in current main in adios2-feedstock
